src/helm-util.ts

```diff
@@ -23,7 +23,12 @@ export interface NameValuePair {
 export function getHelmDownloadURL(version: string): string {
     switch (os.type()) {
         case 'Linux':
-            return util.format('https://get.helm.sh/helm-%s-linux-amd64.zip', version);
+            switch (os.arch()) {
+                case 'arm64':
+                    return util.format('https://get.helm.sh/helm-%s-linux-arm64.zip', version);
+                case 'x64':
+                    return util.format('https://get.helm.sh/helm-%s-linux-amd64.zip', version);
```

Comment:
We should either make 'x64' the default here or throw an error to prevent the switch statement from falling through.

```TypeScript
default:
       throw Error("Unsupported OS architecture");
```

Reply:
Will do. It wil be x64